src/Components/Pages/Details.js

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useState } from 'react';
+import '../../styles/details.css';
+import Table from 'react-bootstrap/Table';
+import { useNavigate, useParams } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { getPollutionData } from '../../redux/slice';
+import Loading from '../../imgs/loading.gif';
+import data from '../../data/data';
+import NavBar from '../props/NavBar';
+import Footer from '../props/Footer';
+import back from '../../imgs/back.png';
+
+function Details() {
+  const { country } = useParams();
+  const [map, setMap] = useState('');
+  const [name, setName] = useState('');
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  useEffect(() => {
```

Comment:
- [ ] The details page should follow the given design as this is a requirement.

Given design
![image](https://user-images.githubusercontent.com/81932478/203498448-6ae51e71-bf29-4faf-83d4-e7a98303894b.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/203498507-0c4ac15c-de59-4706-9187-02b3221216c2.png)

#### Note that even though the color for this design is pink, the layout should be followed with the design that you have chosen